pkg/cli/group_output_test.go

```diff
@@ -155,7 +155,7 @@ func Test_TripleGroupOutput(t *testing.T) {
 		)
 		cli := Init(
 			WithFinder(fsFinder),
-			WithReporter(stdoutReporter),
+			WithReporters([]reporter.Reporter{stdoutReporter}),
```

Comment:
Maybe you could keep and update WithReporter to return a slice of Reporter with one element


It would help you to keep the change simpler, but also to be easier to read.

Reply:
Thinking of changing WithReporters signature to use spread

```go
func WithReporters(r ...reporter.Reporter) Option {}

// Usage
cli.WithReporters(chosenReporters...)
cli.WithReporters(standardReporter)
```

Reply:
It would be a better things, yes